### ISSUE TITLE
[Feature Request] Customizable Privacy Policy Tab

### ISSUE THREAD

**Issue by Techlogist** [osTicket/osTicket#5142]
Hello dear forum,

I am serching the internet for a few days and modifying the OST code, trying to find a way to add a "Privacy Policy" tab to the main landing page:

![Capture](https://user-images.githubusercontent.com/57347209/68110552-86931080-feed-11e9-8968-f5f654c02a7d.PNG)

I tried to use the "Admin Panel/Manage/Pages" with a new page with the type "Other" but it kept giving me an error "File not found.".

I sort of found a way, but it got me thinking that this is a feature worth adding to the Admin Panel for european union users, that can be enabled, customized and disabled at will.

![Capture2](https://user-images.githubusercontent.com/57347209/68110553-86931080-feed-11e9-8522-966769b24022.PNG)

It could have 3 features options:

    Enable the privacy policy
    Text field to enter the privacy policy text

**Comment by ghost** [comment]
Create a page in the other category. 
The page appears on client side ...
![screenshot](https://user-images.githubusercontent.com/49860071/68119217-6588ea80-ff02-11e9-966e-654da9bae0c9.png)

So ... its all buid in and no coding needed (y)

perhaps the devs can create a page category witch create links in the footer and not like picture above.

More links in manage-bar could get out off screen. I had this issue on bad and long translations in this section. Be carefull with additions under manage-tab

Some thoughts 🙂

**Comment by Techlogist** [comment]
Hey Bernard,

Thank you for your reply, I tried that earlier, and mentioned in my first post this _"I tried to use the "Admin Panel/Manage/Pages" with a new page with the type "Other" but it kept giving me an error "File not found."_

I cannot see the reason why it doesnt work, I tried in  couple of VMs with Apache, and reinstalling it a couple of times.

Thanks for the advice thoe :)

**Comment by ghost** [comment]
Stange .. ive got this set up using a fresh install . It should work. 
Got a .htaccess with addtional code on in the root of your install witch might screw some things up ?

**Comment by Techlogist** [comment]
Not really, all standard with that. I already opened a ticket with my hosting provider to to discuss that problem.

**Comment by Techlogist** [comment]
Hey,

Kinda find out how to make it happen . I will fork it and had the mods
